charts/trino/values.yaml

```
@@ -24,6 +24,7 @@ server:
     query:
       maxMemory: "4GB"
       maxMemoryPerNode: "1GB"
+      # maxTotalMemoryPerNode is only used for Trino versions prior to 369.
       maxTotalMemoryPerNode: "2GB"
```

```suggestion
      maxTotalMemoryPerNode: "2GB"  # maxTotalMemoryPerNode is only used for Trino versions prior to 369.
```

Frigate (https://frigate.readthedocs.io/en/latest/customizing.html) that we use for generating docs uses comments on the same line to populate the description column in the chart's docs (https://github.com/trinodb/charts/blob/main/charts/trino/README.md).

---

@erikcw Can you also address this (no need for a new commit, squash with existing)? Seems the only remaining item. I'll merge once done.

---

No problem.